app/controllers/catalog_controller.rb

```diff
@@ -1422,7 +1422,7 @@ def rearrange_groups_array
 
   def get_available_resources(kls)
     @edit[:new][:available_resources] = {}
-    kls.constantize.all.each do |r|
+    kls.constantize.where("type is null or type != 'ServiceTemplateAnsiblePlaybook'").each do |r|
```

Comment:
@kbrock is the the preferred query?  We can fix in follow-up PR - just drawing your attention here.